packages/client/src/services/credit/creditRfqs.ts

```diff
@@ -166,40 +171,32 @@ export const creditRfqsById$ = creditRfqUpdates$.pipe(
             },
           ]
         }
-        // Currently the server sets the state of all the other non-passed quotes to
-        // Rejected if one quote is accepted
         case QUOTE_ACCEPTED_RFQ_UPDATE: {
-          const previousRfq = Object.values(rec).find((rfqDetails) =>
-            rfqDetails.quotes.some((quote) => quote.id === update.payload.id),
-          )
-          if (previousRfq) {
-            return [
-              acc[0],
-              {
-                ...rec,
-                [previousRfq.id]: {
-                  ...previousRfq,
-                  quotes: previousRfq.quotes.map((quote) => ({
-                    ...quote,
-                    state: getQuoteStateOnAccept(quote, update.payload),
-                  })),
-                },
+          const previousRfq = rfqs[update.payload.rfqId]
```

Comment:
like all the other quote updates .. not sure why that slightly convoluted way of grabbing the prev RFQ was used just for the Accept